features/git-town/configuration/non_feature_branches/modify_non_feature_branch_configuration.feature

```diff
@@ -40,19 +58,30 @@ Feature: modifying the non-feature branch configuration
   Scenario: removing a non-feature branch that exists
     Given my non-feature branches are "staging" and "qa"
     When I run `git town non-feature-branches --remove staging`
-    Then I see "Removed 'staging' from non-feature branches"
+    Then I see no output
     And the non-feature branches don't include "staging"
     And the non-feature branches include "qa"
 
 
   Scenario: using invalid parameter
     Given my non-feature branches are "staging" and "qa"
-    When I run `git town non-feature-branches --invalid-parameter staging`
-    Then I see "usage: git town non-feature-branches (--add | --remove) <branchname>"
+    When I run `git town non-feature-branches --invalid-parameter staging` while allowing errors
+    Then I see
+      """
+
+        Error
+        unknown option: --invalid-parameter
+
+      """
 
 
   Scenario: missing branch name
     Given my non-feature branches are "staging" and "qa"
-    When I run `git town non-feature-branches --add`
-    Then I see "Missing branch name"
-    And I see "usage: git town non-feature-branches (--add | --remove) <branchname>"
+    When I run `git town non-feature-branches --add` while allowing errors
+    Then I see
+      """
+
+        Error
+        branch name required
```

Comment:
I don't think so. In this case the user typed `git town non-feature-branches --add` or `git town non-feature-branches --remove`